

<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>Glossary &mdash; GeneLeaf 1 documentation</title>
  

  
  <link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="_static/pygments.css" type="text/css" />

  
  
  
  

  
  <!--[if lt IE 9]>
    <script src="_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="./" src="_static/documentation_options.js"></script>
        <script src="_static/jquery.js"></script>
        <script src="_static/underscore.js"></script>
        <script src="_static/doctools.js"></script>
        <script src="_static/language_data.js"></script>
        <script src="_static/js/gtag.js"></script>
    
    <script type="text/javascript" src="_static/js/theme.js"></script>

    
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="index.html" class="icon icon-home" alt="Documentation Home"> GeneLeaf
          

          
          </a>

          
            
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        
        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <p class="caption"><span class="caption-text">Quick Start Guides:</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="QuickStart/Getting_Started.html">Getting Started</a></li>
</ul>
<p class="caption"><span class="caption-text">Manual:</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="Manual/010-Introduction.html">1. Introduction</a></li>
<li class="toctree-l1"><a class="reference internal" href="Manual/030-Pedigrees.html">2. Pedigree Canvas</a></li>
<li class="toctree-l1"><a class="reference internal" href="Manual/035-Modes.html">3. Modes</a></li>
<li class="toctree-l1"><a class="reference internal" href="Manual/040-Menus.html">4. Menus</a></li>
<li class="toctree-l1"><a class="reference internal" href="Manual/050-Toolbars.html">5. Toolbars</a></li>
<li class="toctree-l1"><a class="reference internal" href="Manual/060-KeyboardCommands.html">6. Keyboard Commands</a></li>
<li class="toctree-l1"><a class="reference internal" href="Manual/070-DataFormat.html">7. Data Format</a></li>
<li class="toctree-l1"><a class="reference internal" href="Manual/080-ImportExport.html">8. Import and Export</a></li>
<li class="toctree-l1"><a class="reference internal" href="Manual/090-DataAnalysis.html">9. Data Analysis</a></li>
<li class="toctree-l1"><a class="reference internal" href="Manual/Data_Model.html">10. Data Model</a></li>
<li class="toctree-l1"><a class="reference internal" href="Manual/Keyboard_Shortcuts.html">11. Keyboard Shortcuts</a></li>
</ul>
<p class="caption"><span class="caption-text">Release Notes:</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="Release_Notes/Beta.html">Pre-Release</a></li>
<li class="toctree-l1"><a class="reference internal" href="Release_Notes/Version1.0.html">Version 1.0 (Spring 2021)</a></li>
<li class="toctree-l1"><a class="reference internal" href="License.html">License</a></li>
</ul>

            
          
        </div>
        
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">GeneLeaf</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="index.html" class="icon icon-home"></a> &raquo;</li>
        
      <li>Glossary</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
            <a href="_sources/Glossary.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="glossary">
<h1>Glossary<a class="headerlink" href="#glossary" title="Permalink to this headline">¶</a></h1>
<dl class="glossary simple">
<dt id="term-allele">allele</dt><dd><p>One of two or more DNA sequences occurring at a particular gene locus. Typically one allele (“normal” DNA sequence) is common, and other alleles (mutations) are rare.</p>
</dd>
<dt id="term-allele-heterogeneity">allele heterogeneity</dt><dd><p>Different mutations in the same gene that cause different phenotypic manifestations or severity of disease.</p>
</dd>
<dt id="term-analysis">analysis</dt><dd><p>A GIMS analysis refers to the results of data analysis for CGS testing.</p>
</dd>
<dt id="term-aneuploidy">aneuploidy</dt><dd><p>The occurrence of one or more extra or missing chromosomes leading to an unbalanced chromosome complement, or any chromosome number that is not an exact multiple of the haploid number (which is 23).</p>
</dd>
<dt id="term-autosomal">autosomal</dt><dd><p>Refers to any of the chromosomes numbered 1-22 or the genes on chromosomes 1-22. This term excludes the sex-determining chromosomes, X and Y.</p>
</dd>
<dt id="term-autosomal-dominant">autosomal dominant</dt><dd><p>Autosomal dominant inheritance refers to genetic conditions that occur when a mutation is present in one copy of a given gene (i.e., the person is heterozygous).</p>
</dd>
<dt id="term-autosomal-recessive">autosomal recessive</dt><dd><p>Autosomal recessive inheritance refers to genetic conditions that occur only when mutations are present in both copies of a given gene (i.e., the person is homozygous for a mutation, or carries two different mutations of the same gene, a state referred to as compound heterozygosity).</p>
</dd>
<dt id="term-base-pair">base pair</dt><dd><p>Two nitrogen-containing bases pair together between double-stranded DNA; only specific combinations of these bases (e.g., adenine with thymine; guanine with cytosine) are possible, a fact which facilitates accurate DNA replication; when quantified (e.g., 8 base pairs, or bp), this term refers to the actual number of base pairs in a sequence of nucleotides.</p>
</dd>
<dt id="term-biallelic">biallelic</dt><dd><p>Of or pertaining to both alleles of a single gene (paternal and maternal). For example, biallelic mutation carriers have a mutation (not necessarily the same mutation) in both copies of a particular gene (a paternal and a maternal mutation).</p>
</dd>
<dt id="term-Bioinformatics">Bioinformatics</dt><dd><p>Bioinformatics is an interdisciplinary field that develops methods and software tools for understanding biological data. As an interdisciplinary field of science, bioinformatics combines computer science, statistics, mathematics, and engineering to analyze and interpret biological data. Bioinformatics has been used for in silico analyses of biological queries using mathematical and statistical techniques.</p>
</dd>
<dt id="term-carrier">carrier</dt><dd><p>In classical genetics, an individual who carries one deleterious allele for an autosomal recessive disorder. In clinical discussions, may refer to an individual who carries a deleterious allele that predisposes to disease.</p>
</dd>
<dt id="term-carrier-frequency">carrier frequency</dt><dt id="term-carrier-rate">carrier rate</dt><dd><p>The proportion of individuals in a population who have a single copy of a specific recessive gene mutation; also sometimes applied to the prevalence of mutations in dominantly acting genes such as BRCA1 and BRCA2. Also called carrier rate.</p>
</dd>
<dt id="term-chromosome">chromosome</dt><dd><p>Discrete physical structures inside a cell nucleus that consist of proteins and DNA organized into genes.</p>
</dd>
<dt id="term-clone">clone</dt><dd><p>An identical copy of a DNA sequence or entire gene; one or more cells derived from and identical to a single ancestor cell OR to isolate a gene or specific sequence of DNA.</p>
</dd>
<dt id="term-CNV">CNV</dt><dt id="term-copy-number-variant">copy number variant</dt><dd><p>Refers to the genetic trait involving the number of copies of a particular gene present in the genome of an individual. Genetic variants, including insertions, deletions, and duplications of segments of DNA, are also collectively referred to as CNVs. CNVs account for a significant proportion of the genetic variation between individuals. Also called copy number variant.</p>
</dd>
<dt id="term-CODIS">CODIS</dt><dd><p>CODIS is the acronym for the Combined DNA Index System and is the generic term used to describe the FBI’s program of support for criminal justice DNA databases as well as the software used to run these databases. Prior to January 1, 2017, the national level of CODIS required that known offender profiles have a set of 13 loci called the “CODIS core”. Since then, the requirement has expanded to include seven additional loci. Partial profiles are also allowed in CODIS in separate indexes and are common in crime scene samples that are degraded or are mixtures of multiple individuals. Upload of these profiles to the national level of CODIS requires at least eight of the core loci to be present as well as a profile rarity of 1 in 10 million (calculated using population statistics).</p>
</dd>
<dt id="term-codon">codon</dt><dd><p>In DNA or RNA, a sequence of 3 consecutive nucleotides that codes for a specific amino acid or signals the termination of gene translation (stop or termination codon).
compound heterozygosity
The presence of two different mutated alleles at a particular gene locus.</p>
</dd>
<dt id="term-Compound-heterozygosity">Compound heterozygosity</dt><dd><p>Compound heterozygosity in medical genetics is the condition of having two heterogeneous recessive alleles at a particular locus that can cause genetic disease in a heterozygous state. That is, an organism is a compound heterozygote when it has two recessive alleles for the same gene, but with those two alleles being different from each other (for example, both alleles might be mutated but at different locations). Compound heterozygosity reflects the diversity of the mutation base for many autosomal recessive genetic disorders; mutations in most disease-causing genes have arisen many times. This means that many cases of disease arise in individuals who have two unrelated alleles, who technically are heterozygotes, but both the alleles are defective.</p>
</dd>
<dt id="term-congenital">congenital</dt><dd><p>A condition or trait present at birth. It may be the result of genetic or non-genetic factors.</p>
</dd>
<dt id="term-consanguinity">consanguinity</dt><dd><p>Genetic relatedness between individuals who are descendants of at least one common ancestor.</p>
</dd>
<dt id="term-constitutional-DNA">constitutional DNA</dt><dd><p>Constitutional DNA refers to tissue derived from reproductive cells (egg or sperm) that become incorporated into the DNA of every cell in the body of the offspring. A germline mutation may be passed from parent to offspring. Also called germline DNA.</p>
</dd>
<dt id="term-Container">Container</dt><dd><p>A GIMS container is a laboratory container (e.g., a tube or plate), which
contains one or more samples.</p>
</dd>
<dt id="term-cosegregation">cosegregation</dt><dd><p>The transmission, together, of 2 or more genes on the same chromosome, as a result of their being in very close physical proximity to one another (i.e., linked).</p>
</dd>
<dt id="term-cytogenetics">cytogenetics</dt><dd><p>The study of the structure, function, and abnormalities of human chromosomes.</p>
</dd>
<dt id="term-de-novo-variant">de novo variant</dt><dt id="term-de-novo-mutation">de novo mutation</dt><dd><p>A genetic alteration that is present for the first time in one family member as a result of a variant (or mutation) in a germ cell (egg or sperm) of one of the parents, or a variant that arises in the fertilized egg itself during early embryogenesis. Also called de novo mutation, new mutation, and new variant.</p>
</dd>
<dt id="term-deleterious-mutation">deleterious mutation</dt><dd><p>A genetic alteration that increases an individual’s susceptibility or predisposition to a certain disease or disorder. When such a variant (or mutation) is inherited, development of symptoms is more likely, but not certain. Also called disease-causing mutation, pathogenic variant, predisposing mutation, and susceptibility gene.</p>
</dd>
<dt id="term-deletion">deletion</dt><dd><p>A type of genetic change that involves the absence of a segment of DNA. It may be as small as a single base but can vary significantly in size.</p>
</dd>
<dt id="term-depth-of-coverage">depth of coverage</dt><dd><p>Refers to the number of times a nucleotide is read during sequencing. A greater depth of coverage can increase confidence in the final results. Deep coverage aids in differentiating sequencing errors from single nucleotide polymorphisms. This can be specifically useful when a patient has a mosaicism or when a tumor is heterogeneous for a mutation.</p>
</dd>
<dt id="term-disease-causing-mutation">disease-causing mutation</dt><dd><p>A genetic alteration that increases an individual’s susceptibility or predisposition to a certain disease or disorder. When such a variant (or mutation) is inherited, development of symptoms is more likely, but not certain. Also called deleterious mutation, pathogenic variant, predisposing mutation, and susceptibility gene.</p>
</dd>
<dt id="term-DNA">DNA</dt><dt id="term-deoxyribonucleic-acid">deoxyribonucleic acid</dt><dd><p>The molecular basis of heredity; encodes the genetic information responsible for the development and function of an organism and allows for transmission of that genetic information from one generation to the next. The DNA molecule is structured as a double-stranded helix held together by weak hydrogen bonds between purine-pyrimidine nucleotide base pairs: adenine (A) paired with thymine (T), and guanine (G) paired with cytosine (C). Also called deoxyribonucleic acid.</p>
</dd>
<dt id="term-domain">domain</dt><dd><p>A specific physical region or amino acid sequence in a protein which is associated with a particular function or corresponding segment of DNA.</p>
</dd>
<dt id="term-double-heterozygosity">double heterozygosity</dt><dd><p>The presence of two different mutated alleles at two separate genetic loci.</p>
</dd>
<dt id="term-epigenetics">epigenetics</dt><dd><p>The study of heritable changes that do not affect the DNA sequence but influence gene expression.</p>
</dd>
<dt id="term-epimutation">epimutation</dt><dt id="term-epigenetic-variant">epigenetic variant</dt><dt id="term-epigenetic-alteration">epigenetic alteration</dt><dd><p>A heritable change that does not affect the DNA sequence but results in a change in gene expression. Examples include promoter methylation and histone modifications. Also called epigenetic alteration and epigenetic variant.</p>
</dd>
<dt id="term-ER-negative-PR-negative-HER2-neu-negative-breast-cancer">ER-negative PR-negative HER2/neu-negative breast cancer</dt><dd><p>ER-negative PR-negative HER2/neu-negative breast cancer is defined by a lack of expression of estrogen receptor (ER), progesterone receptor (PR), and human epidermal growth factor receptor 2 (HER2/neu). Also called <strong>triple-negative breast cancer</strong>.</p>
</dd>
<dt id="term-euchromatin">euchromatin</dt><dd><p>Euchromatin is a lightly packed form of chromatin (DNA, RNA and protein) that is enriched in genes, and is often (but not always) under active transcription. Euchromatin comprises the most active portion of the genome within the cell nucleus. 92% of the human genome is euchromatic. The remainder is called heterochromatin.</p>
</dd>
<dt id="term-exome">exome</dt><dd><p>The exome is the part of the genome formed by exons, the sequences which when transcribed remain within the mature RNA after introns are removed by RNA splicing. It consists of all DNA that is transcribed into mature RNA in cells of any type as distinct from the transcriptome, which is the RNA that has been transcribed only in a specific cell population. The exome of the human genome consists of roughly 180,000 exons constituting about 1% of the total genome, or about 30 megabases of DNA. Though comprising a very small fraction of the genome, mutations in the exome are thought to harbor 85% of mutations that have a large effect on disease. Exome sequencing has proved to be an efficient strategy to determine the genetic basis of more than two dozen Mendelian or single gene disorders.</p>
</dd>
<dt id="term-exon">exon</dt><dd><p>The sequence of DNA present in mature messenger RNA, some of which encodes the amino acids of a protein. Most genes have multiple exons with introns between them.</p>
</dd>
<dt id="term-facies">facies</dt><dd><p>A distinctive facial feature or expression that is characteristic of a specific condition.</p>
</dd>
<dt id="term-false-positive-result">false-positive result</dt><dd><p>A test result that indicates an individual is affected and/or has a certain gene mutation when he or she is actually unaffected and/or does not have the mutation; i.e., a positive test result in a truly unaffected or mutation-negative individual.</p>
</dd>
<dt id="term-familial">familial</dt><dd><p>A phenotype or trait that occurs with greater frequency in a given family than in the general population; familial traits may have a genetic and/or nongenetic etiology.</p>
</dd>
<dt id="term-Family">Family</dt><dd><p>A GIMS family is a record of the genetic relationship between individuals. For example, the proband as well as the proband’s parents. In many cases, family information is captured within an order (i.e., a trio order contains the proband as well as the parents of the proband).</p>
</dd>
<dt id="term-family-history">family history</dt><dt id="term-family-medical-history">family medical history</dt><dd><p>The genetic relationships within a family combined with the medical history of individual family members. When represented in diagram form using standardized symbols and terminology, it is usually referred to as a pedigree or family tree. Also called family medical history.</p>
</dd>
<dt id="term-FDR">FDR</dt><dt id="term-first-degree-relative">first-degree relative</dt><dd><p>The parents, siblings, or children of an individual. Also called first-degree relative.</p>
</dd>
<dt id="term-FISH">FISH</dt><dt id="term-fluorescence-in-situ-hybridization">fluorescence in situ hybridization</dt><dd><p>A technique used to identify the presence of specific chromosomes or chromosomal regions through hybridization (attachment) of fluorescently-labeled DNA probes to denatured chromosomal DNA. Examination through a microscope under fluorescent lighting detects the presence of the colored hybridized signal (and hence presence of the chromosome material) or absence of the hybridized signal (and hence absence of the chromosome material). Also called fluorescence in situ hybridization.</p>
</dd>
<dt id="term-founder-variant">founder variant</dt><dt id="term-founder-mutation">founder mutation</dt><dd><p>A genetic alteration observed with high frequency in a group that is or was geographically or culturally isolated, in which one or more of the ancestors was a carrier of the altered gene. This phenomenon is often called a founder effect. Also called founder mutation.</p>
</dd>
<dt id="term-frameshift-variant">frameshift variant</dt><dt id="term-frameshift-mutation">frameshift mutation</dt><dd><p>An insertion or deletion involving a number of base pairs that is not a multiple of three, which consequently disrupts the triplet reading frame of a DNA sequence. Such variants (or mutations) usually lead to the creation of a premature termination (stop) codon, and result in a truncated (shorter-than-normal) protein product. Also called frameshift variant.</p>
</dd>
<dt id="term-gene">gene</dt><dd><p>The basic unit of heredity that occupies a specific location on a chromosome. Each consists of nucleotides arranged in a linear manner. Most genes code for a specific protein or segment of protein leading to a particular characteristic or function.</p>
</dd>
<dt id="term-genetic-anticipation">genetic anticipation</dt><dd><p>A phenomenon in which the signs and symptoms of some genetic conditions tend to become more severe and/or appear at an earlier age as the disorder is passed from one generation to the next. Huntington disease is an example of a genetic disorder in which the biological mechanism for this phenomenon has been well documented. In other cases, it may be due to factors such as increased surveillance or other nongenetic causes.</p>
</dd>
<dt id="term-genetic-heterogeneity">genetic heterogeneity</dt><dd><p>The production of the same or similar phenotypes (observed biochemical, physiological, and morphological characteristics of a person determined by his/her genotype) by different genetic mechanisms. There are two types: (1) allelic heterogeneity – when different alleles at a locus can produce variable expression of a condition; and (2) locus heterogeneity – the term used to describe disease in which mutations at different loci can produce the same disease phenotype.</p>
</dd>
<dt id="term-genetic-marker">genetic marker</dt><dd><p>An identifiable segment of DNA (e.g., Single Nucleotide Polymorphism [SNP], Restriction Fragment Length Polymorphism [RFLP], Variable Number of Tandem Repeats [VNTR], microsatellite) with enough variation between individuals that its inheritance and co-inheritance with alleles of a given gene can be traced; used in linkage analysis.</p>
</dd>
<dt id="term-genetic-predisposition">genetic predisposition</dt><dt id="term-genetic-susceptibility">genetic susceptibility</dt><dd><p>Increased likelihood or chance of developing a particular disease due to the presence of one or more gene mutations and/or a family history that indicates an increased risk of the disease. Also called genetic susceptibility.</p>
</dd>
<dt id="term-genetic-screening">genetic screening</dt><dd><p>Genetic testing designed to identify individuals in a given population who are at higher risk of having or developing a particular disorder, or carrying a gene for a particular disorder.</p>
</dd>
<dt id="term-genetic-variant">genetic variant</dt><dd><p>An alteration in the most common DNA nucleotide sequence. The term variant can be used to describe an alteration that may be benign, pathogenic, or of unknown significance. The term variant is increasingly being used in place of the term mutation.</p>
</dd>
<dt id="term-genomic-imprinting">genomic imprinting</dt><dd><p>An epigenetic process resulting in the inactivation of an allele depending on which parent it was inherited from. Genomic imprinting can have clinical relevance because it may affect the expression of a gene mutation (i.e., the phenotype) in the offspring of an affected parent depending on which parent is passing on the mutation.</p>
</dd>
<dt id="term-genotype">genotype</dt><dd><p>At its broadest level, genotype includes the entire genetic constitution of an individual. It is often applied more narrowly to the set of alleles present at one or more specific loci.</p>
</dd>
<dt id="term-germline">germline</dt><dd><p>The cells from which eggs or sperm (i.e., gametes) are derived.</p>
</dd>
<dt id="term-germline-DNA">germline DNA</dt><dd><p>Germline DNA refers to tissue derived from reproductive cells (egg or sperm) that become incorporated into the DNA of every cell in the body of the offspring. A germline mutation may be passed from parent to offspring. Also called constitutional DNA.</p>
</dd>
<dt id="term-germline-variant">germline variant</dt><dd><p>A gene change in a reproductive cell (egg or sperm) that becomes incorporated into the DNA of every cell in the body of the offspring. A variant contained within the germline can be passed from parent to offspring, and is, therefore, hereditary.</p>
</dd>
<dt id="term-GWAS">GWAS</dt><dt id="term-genome-wide-association-study">genome-wide association study</dt><dd><p>A genome-wide association study (GWAS) is a way for scientists to identify inherited genetic variants associated with risk of disease or a particular trait. This method surveys the entire genome for genetic polymorphisms, typically single nucleotide polymorphisms (SNPs) (pronounced “snips”), that occur more frequently in cases (people with the disease or trait being assessed) than in controls (people without the disease or trait). Also called GWAS.</p>
</dd>
<dt id="term-haploinsufficiency">haploinsufficiency</dt><dd><p>The situation that occurs when one copy of a gene is inactivated or deleted and the remaining functional copy of the gene is not adequate to produce the needed gene product to preserve normal function.</p>
</dd>
<dt id="term-haplotype">haplotype</dt><dd><p>A set of closely linked genetic markers present on one chromosome which tend to be inherited together.</p>
</dd>
<dt id="term-hemizygous">hemizygous</dt><dd><p>Describes an individual who has only one member of a chromosome pair or chromosome segment rather than the usual two. Hemizygosity is often used to describe X-linked genes in males who have only one X chromosome. This term is sometimes used in somatic cell genetics where cancer cell lines are often hemizygous for certain alleles or chromosomal regions.</p>
</dd>
<dt id="term-heritability">heritability</dt><dd><p>The proportion of variation in a population trait that can be attributed to inherited genetic factors. Heritability estimates range from 0 to 1 and are often expressed as a percentage. A number close to 1 may be indicative of a highly heritable trait within a population. It should not be used to estimate risk on an individual basis.</p>
</dd>
<dt id="term-heterochromatin">heterochromatin</dt><dd><p>Heterochromatin is a tightly packed form of DNA, which comes in multiple varieties. These varieties lie on a continuum between the two extremes of constitutive and facultative heterochromatin.</p>
</dd>
<dt id="term-heteroduplex-analysis">heteroduplex analysis</dt><dd><p>A method of detecting sequence differences between normal DNA and the DNA to be tested. It is commonly used as a screening method to detect potential mutations in a gene.</p>
</dd>
<dt id="term-heterozygous-genotype">heterozygous genotype</dt><dd><p>Occurs when the two alleles at a particular gene locus are different. A heterozygous genotype may include one normal allele and one mutation, or two different mutations. The latter is called a compound heterozygote.</p>
</dd>
<dt id="term-HLOD-score">HLOD score</dt><dt id="term-heterogeneity-logarithm-of-the-odds-score">heterogeneity logarithm of the odds score</dt><dd><p>A statistical estimate of whether two genetic loci are physically near enough to each other (or “linked”) on a particular chromosome that they are likely to be inherited together. A heterogeneity logarithm of the odds score is calculated in the presence of locus heterogeneity (when the same phenotype can be caused by mutations in genes at different chromosomal loci). Also called HLOD score.</p>
</dd>
<dt id="term-homozygous-genotype">homozygous genotype</dt><dd><p>Occurs when both alleles at a particular gene locus are the same. A person may be homozygous for the normal allele or for a mutation.</p>
</dd>
<dt id="term-index-case">index case</dt><dd><p>A clinically affected individual through whom attention is first drawn to a genetic disorder in a family.</p>
</dd>
<dt id="term-informative">informative</dt><dd><p>In genetic testing, a test result that reveals definitively the presence or absence of the germline genetic alteration associated with the hereditary disorder being assessed. In linkage analysis, the ability to distinguish between maternally inherited and paternally inherited DNA markers (polymorphisms) within or near a given gene of interest.</p>
</dd>
<dt id="term-inherited-cancer-syndrome">inherited cancer syndrome</dt><dd><p>Describes the clinical manifestations associated with a mutation conferring cancer susceptibility.</p>
</dd>
<dt id="term-insertion">insertion</dt><dd><p>A type of genetic change that involves the addition of a segment of DNA. It may be as small as a single base but can vary significantly in size.</p>
</dd>
<dt id="term-intron">intron</dt><dd><p>The sequence of DNA in between exons that is initially copied into RNA but is cut out of the final RNA transcript and therefore does not change the amino acid code. Some intronic sequences are known to affect gene expression.</p>
</dd>
<dt id="term-inversion">inversion</dt><dd><p>A chromosomal defect in which a segment of the chromosome breaks off and reattaches in the reverse direction.</p>
</dd>
<dt id="term-kindred">kindred</dt><dd><p>An extended family.</p>
</dd>
<dt id="term-late-or-variable-onset">late or variable onset</dt><dd><p>The state in which a genetic trait is expressed later in life or is expressed at no fixed time in a life history.</p>
</dd>
<dt id="term-LD">LD</dt><dd><p>Where alleles (DNA markers) occur together more often than can be accounted for by chance because of their physical proximity on a chromosome. Also called linkage disequilibrium.</p>
</dd>
<dt id="term-linkage">linkage</dt><dd><p>The tendency for genes or segments of DNA closely positioned along a chromosome to segregate together at meiosis, and therefore be inherited together.</p>
</dd>
<dt id="term-linkage-analysis">linkage analysis</dt><dd><p>A gene-hunting technique that traces patterns of disease in high-risk families. It attempts to locate a disease-causing gene by identifying genetic markers of known chromosomal location that are co-inherited with the trait of interest.</p>
</dd>
<dt id="term-linkage-disequilibrium">linkage disequilibrium</dt><dd><p>Laboratory Information System - a system used for tracking and storing laboratory sample information (the term LIS is generally used in clinical contexts, whereas the term LIMS may be used in clinical or non-clinical contexts)</p>
</dd>
<dt id="term-locus">locus</dt><dd><p>The physical site or location of a specific gene on a chromosome.</p>
</dd>
<dt id="term-locus-heterogeneity">locus heterogeneity</dt><dd><p>The same phenotype is caused by mutations in genes at different chromosomal loci.</p>
</dd>
<dt id="term-microsatellite">microsatellite</dt><dd><p>Repetitive segments of DNA scattered throughout the genome in noncoding regions between genes or within genes (introns). They are often used as markers for linkage analysis because of their naturally occurring high variability in repeat number between individuals. These regions are inherently genetically unstable and susceptible to mutations.</p>
</dd>
<dt id="term-microsatellite-instability">microsatellite instability</dt><dd><p>A characteristic of cells that contain an abnormality in DNA mismatch repair (see microsatellite). For example, the presence of MSI in colorectal tumor tissue may be used as a marker for germline mutations in one of the DNA mismatch repair genes associated with HNPCC. MSI can also occur sporadically, and in these cases is related to gene hypermethylation. This is an issue in the differential diagnosis of HNPCC. Also called MSI.</p>
</dd>
<dt id="term-missense-mutation">missense mutation</dt><dd><p>A genetic alteration in which a single base pair substitution alters the genetic code in a way that produces an amino acid that is different from the usual amino acid at that position. Some missense variants (or mutations) will alter the function of the protein. Also called missense variant.</p>
</dd>
<dt id="term-missense-variant">missense variant</dt><dd><p>A genetic alteration in which a single base pair substitution alters the genetic code in a way that produces an amino acid that is different from the usual amino acid at that position. Some missense variants (or mutations) will alter the function of the protein. Also called missense mutation.</p>
</dd>
<dt id="term-mitochondiral-DNA">mitochondiral DNA</dt><dd><p>Mitochondrial DNA (mtDNA or mDNA) is the DNA located in mitochondria, cellular organelles within eukaryotic cells that convert chemical energy from food into a form that cells can use, adenosine triphosphate (ATP). Mitochondrial DNA is only a small portion of the DNA in a eukaryotic cell; most of the DNA can be found in the cell nucleus. In humans, the 16,569 base pairs of mitochondrial DNA encode for only 37 genes. Human mitochondrial DNA was the first significant part of the human genome to be sequenced. In most species, including humans, mtDNA is inherited solely from the mother.</p>
</dd>
<dt id="term-MLPA">MLPA</dt><dd><p>A laboratory method commonly used for the detection of unusual copy number changes (insertions or deletions) of genomic sequences. Also called multiplex ligation-dependent probe amplification.</p>
</dd>
<dt id="term-mode-of-inheritance">mode of inheritance</dt><dd><p>The manner in which a genetic trait or disorder is passed from one generation to the next. Autosomal dominant, autosomal recessive, X-linked dominant, X-linked recessive, multifactorial, and mitochondrial inheritance are examples. Each mode of inheritance results in a characteristic pattern of affected and unaffected family members.</p>
</dd>
<dt id="term-mosaicism">mosaicism</dt><dd><p>The occurrence of 2 or more cell lines with different genetic or chromosomal make-up, within a single individual or tissue.</p>
</dd>
<dt id="term-MSI">MSI</dt><dd><p>A characteristic of cells that contain an abnormality in DNA mismatch repair (see microsatellite). For example, the presence of MSI in colorectal tumor tissue may be used as a marker for germline mutations in one of the DNA mismatch repair genes associated with HNPCC. MSI can also occur sporadically, and in these cases is related to gene hypermethylation. This is an issue in the differential diagnosis of HNPCC. Also called microsatellite instability.</p>
</dd>
<dt id="term-multigene-test">multigene test</dt><dd><p>Genetic tests that use next-generation sequencing to test multiple genes simultaneously. Also called multiple-gene panel test and multiple-gene test.</p>
</dd>
<dt id="term-multiple-gene-panel-test">multiple-gene panel test</dt><dd><p>Genetic tests that use next-generation sequencing to test multiple genes simultaneously. Also called multigene test and multiple-gene test.</p>
</dd>
<dt id="term-multiple-gene-test">multiple-gene test</dt><dd><p>Genetic tests that use next-generation sequencing to test multiple genes simultaneously. Also called multigene test and multiple-gene panel test.</p>
</dd>
<dt id="term-multiplex-genomic-test">multiplex genomic test</dt><dd><p>A method for detecting multiple genetic alterations (i.e., gene mutations or single nucleotide polymorphisms in a single gene or across the genome) simultaneously.</p>
</dd>
<dt id="term-multiplex-ligation-dependent-probe-amplification">multiplex ligation-dependent probe amplification</dt><dd><p>A laboratory method commonly used for the detection of unusual copy number changes (insertions or deletions) of genomic sequences. Also called MLPA.</p>
</dd>
<dt id="term-mutation">mutation</dt><dd><p>A change in the usual DNA sequence at a particular gene locus. Although the term often has a negative connotation, mutations (including polymorphisms) can be harmful, beneficial, or neutral in their effect on cell function. The term variant is sometimes used as a synonym for the term mutation.</p>
</dd>
<dt id="term-mutation-analysis">mutation analysis</dt><dd><p>A germline genetic testing method targeted to detect a specific variant or mutation (such as a deleterious MSH2 variant previously identified in a family), panel of variants (such as the 3 BRCA pathogenic variants comprising the founder mutation panel for individuals of Ashkenazi Jewish ancestry) or type of variant (such as large deletions or insertions in the BRCA1 gene). This type of testing is distinct from complete gene sequencing or variant scanning. The latter are designed to detect most variants in the region being tested. Current usage also applies this term to any genetic test.</p>
</dd>
<dt id="term-negative-predictive-value">negative predictive value</dt><dd><p>The likelihood that an individual with a negative test result is truly unaffected and/or does not have the particular gene mutation in question. Also called NPV.</p>
</dd>
<dt id="term-new-mutation">new mutation</dt><dd><p>A genetic alteration that is present for the first time in one family member as a result of a variant (or mutation) in a germ cell (egg or sperm) of one of the parents, or a variant that arises in the fertilized egg itself during early embryogenesis. Also called de novo mutation, de novo variant, and new variant.</p>
</dd>
<dt id="term-new-variant">new variant</dt><dd><p>A genetic alteration that is present for the first time in one family member as a result of a variant (or mutation) in a germ cell (egg or sperm) of one of the parents, or a variant that arises in the fertilized egg itself during early embryogenesis. Also called de novo mutation, de novo variant, and new mutation.</p>
</dd>
<dt id="term-NGS">NGS</dt><dt id="term-next-generation-sequencing">next generation sequencing</dt><dd><p>A high-throughput method used to determine a portion of the nucleotide sequence of an individual’s genome. This technique utilizes DNA sequencing technologies that are capable of processing multiple DNA sequences in parallel. Also called massively parallel sequencing and next-generation sequencing.</p>
</dd>
<dt id="term-noncarrier">noncarrier</dt><dd><p>An individual who does not carry a mutation previously identified in his or her family.</p>
</dd>
<dt id="term-nonpenetrance">nonpenetrance</dt><dd><p>The state in which a genetic trait, although present in the appropriate genotype, fails to manifest itself in the phenotype (e.g., a woman with a BRCA1 mutation who lives to be elderly and never develops breast or ovarian cancer).</p>
</dd>
<dt id="term-nonsense-mutation">nonsense mutation</dt><dt id="term-nonsense-variant">nonsense variant</dt><dd><p>A genetic alteration that causes the premature termination of a protein. The altered protein may be partially or completely inactivated, resulting in a change or loss of protein function. Also called nonsense variant.</p>
</dd>
<dt id="term-novel-mutation">novel mutation</dt><dt id="term-novel-variant">novel variant</dt><dd><p>A newly discovered, distinct genetic alteration; NOT the same as new or de novo variant (or mutation). Also called novel variant.</p>
</dd>
<dt id="term-NPV">NPV</dt><dd><p>The likelihood that an individual with a negative test result is truly unaffected and/or does not have the particular gene mutation in question. Also called negative predictive value.</p>
</dd>
<dt id="term-nucleotide">nucleotide</dt><dd><p>A molecule consisting of a nitrogen-containing base (adenine, guanine, thymine, or cytosine in DNA; adenine, guanine, uracil, or cytosine in RNA), a phosphate group, and a sugar (deoxyribose in DNA; ribose in RNA). DNA and RNA are polymers comprised of many nucleotides, strung together like beads in a necklace.</p>
</dd>
<dt id="term-null-allele">null allele</dt><dd><p>A mutation that results in either no gene product or the absence of function at the phenotypic level.</p>
</dd>
<dt id="term-nutrigenomics">nutrigenomics</dt><dd><p>The study of the interaction of dietary and genetic factors and its effect on metabolism, health status, and risk of disease.</p>
</dd>
<dt id="term-pathogenic-variant">pathogenic variant</dt><dd><p>A genetic alteration that increases an individual’s susceptibility or predisposition to a certain disease or disorder. When such a variant (or mutation) is inherited, development of symptoms is more likely, but not certain. Also called deleterious mutation, disease-causing mutation, predisposing mutation, and susceptibility gene.</p>
</dd>
<dt id="term-pathognomonic">pathognomonic</dt><dd><p>Findings that are distinctive or characteristic of a particular disease or condition and make the diagnosis.</p>
</dd>
<dt id="term-Patient">Patient</dt><dd><p>A GIMS patient is an individual with a record in the Epic EMR system. Patients may include an individual undergoing testing by the CGS (i.e., the proband) as well as individuals not undergoing testing by the CGS (ie., the proband’s relatives), as well as any other service.</p>
</dd>
<dt id="term-pedigree">pedigree</dt><dd><p>A graphic illustration of family history.</p>
</dd>
<dt id="term-penetrance">penetrance</dt><dd><p>A characteristic of a genotype; it refers to the likelihood that a clinical condition will occur when a particular genotype is present.</p>
</dd>
<dt id="term-phenocopy">phenocopy</dt><dd><p>A phenotypic trait or disease that resembles the trait expressed by a particular genotype, but in an individual who is not a carrier of that genotype. For example, breast cancer in a hereditary breast/ovarian cancer syndrome family member who does not carry the family’s BRCA1 or BRCA2 mutation would be considered a phenocopy. Such an individual does not have the family’s cancer-related mutation and therefore, they do not have the associated cancer risk from that specific mutation.</p>
</dd>
<dt id="term-phenotype">phenotype</dt><dd><p>The observable characteristics in an individual resulting from the expression of genes; the clinical presentation of an individual with a particular genotype.</p>
</dd>
<dt id="term-point-mutation">point mutation</dt><dt id="term-point-variant">point variant</dt><dd><p>A genetic alteration caused by the substitution of a single nucleotide for another nucleotide. Also called point variant.</p>
</dd>
<dt id="term-polymerase-chain-reaction">polymerase chain reaction</dt><dt id="term-PCR">PCR</dt><dd><p>A procedure that produces millions of copies of a short segment of DNA through repeated cycles of: (1) denaturation, (2) annealing, and (3) elongation. PCR is a very common procedure in molecular genetic testing and may be used to generate a sufficient quantity of DNA to perform a test (e.g., allele-specific amplification, trinucleotide repeat quantification). Also called polymerase chain reaction.</p>
</dd>
<dt id="term-polymorphism">polymorphism</dt><dd><p>A common mutation. “Common” is typically defined as an allele frequency of at least 1%. All genes occur in pairs, except when x and y chromosomes are paired in males; thus a polymorphism with an allele frequency of 1% would be found in about 2% of the population, with most carriers having one copy of the polymorphism and one copy of the normal allele.</p>
</dd>
<dt id="term-population-risk">population risk</dt><dd><p>The proportion of individuals in the general population who are affected with a particular disorder or who carry a certain gene; often discussed in the genetic counseling process as a comparison to the patient’s personal risk given his or her family history or other circumstances.</p>
</dd>
<dt id="term-PPV">PPV</dt><dt id="term-positive-predictive-value">positive predictive value</dt><dd><p>The likelihood that an individual with a positive test result truly has the particular gene and/or disease in question. Also called positive predictive value.</p>
</dd>
<dt id="term-predisposing-mutation">predisposing mutation</dt><dd><p>A genetic alteration that increases an individual’s susceptibility or predisposition to a certain disease or disorder. When such a variant (or mutation) is inherited, development of symptoms is more likely, but not certain. Also called deleterious mutation, disease-causing mutation, pathogenic variant, and susceptibility gene.</p>
</dd>
<dt id="term-presymptomatic-testing">presymptomatic testing</dt><dd><p>Genetic analysis of an asymptomatic or unaffected individual who is at risk of a specific genetic disorder.</p>
</dd>
<dt id="term-proband">proband</dt><dd><p>The individual through whom a family with a genetic disorder is ascertained. A proband is the person serving as the starting point for the genetic study of a family.</p>
</dd>
<dt id="term-proposita">proposita</dt><dd><p>The female individual through whom a family with a genetic disorder is ascertained. In males this is called a propositus.</p>
</dd>
<dt id="term-propositus">propositus</dt><dd><p>The male individual through whom a family with a genetic disorder is ascertained. In females this is called a proposita.</p>
</dd>
<dt id="term-pseudogene">pseudogene</dt><dd><p>A DNA sequence that resembles a gene but has been mutated into an inactive form over the course of evolution. It often lacks introns and other essential DNA sequences necessary for function. Though genetically similar to the original functional gene, pseudogenes do not result in functional proteins, although some may have regulatory effects.</p>
</dd>
<dt id="term-recurrence-risk">recurrence risk</dt><dd><p>In genetics, the likelihood that a hereditary trait or disorder present in one family member will occur again in other family members. This is distinguished from recurrence risk for cancer, which is the chance that a cancer that has been treated will recur.</p>
</dd>
<dt id="term-ribosomal-RNA">ribosomal RNA</dt><dd><p>Ribosomal ribonucleic acid (rRNA) is the RNA component of the ribosome, and is essential for protein synthesis in all living organisms. It constitutes the predominant material within the ribosome, which is approximately 60% rRNA and 40% protein by weight, or 3/5 of ribosome mass. Ribosomes contain two major rRNAs and 50 or more proteins. The ribosomal RNAs form two subunits, the large subunit (LSU) and small subunit (SSU). The LSU rRNA acts as a ribozyme, catalyzing peptide bond formation. rRNA sequences are widely used for working out evolutionary relationships among organisms, since they are of ancient origin and are found in all known forms of life.</p>
</dd>
<dt id="term-risk-assessment">risk assessment</dt><dd><p>The quantitative or qualitative assessment of an individual’s risk of carrying a certain gene mutation, or developing a particular disorder, or of having a child with a certain disorder; sometimes done by using mathematical or statistical models incorporating such factors as personal health history, family medical history and ethnic background.</p>
</dd>
<dt id="term-sample">sample</dt><dd><p>A GIMS sample is a laboratory sample, upon which laboratory operations
are to be completed.</p>
</dd>
<dt id="term-sanger-sequencing">sanger sequencing</dt><dd><p>A low-throughput method used to determine a portion of the nucleotide sequence of an individual’s genome. This technique uses polymerase chain reaction (PCR) amplification of genetic regions of interest followed by sequencing of PCR products.</p>
</dd>
<dt id="term-screening">screening</dt><dd><p>Clinical evaluation of an asymptomatic individual in the general population aimed at identifying abnormalities which might signal the presence of a specific medical condition. The intent is to find diseases at the earliest possible stage in their development in order to improve the chances for cure or reduce morbidity.</p>
</dd>
<dt id="term-SDR">SDR</dt><dt id="term-second-degree-relative">second-degree relative</dt><dd><p>The aunts, uncles, grandparents, grandchildren, nieces, nephews, or half-siblings of an individual. Also called <strong>second-degree relative</strong>.</p>
</dd>
<dt id="term-segregation-analysis">segregation analysis</dt><dd><p>The process of fitting formal genetic models to data on expressed disease characteristics (phenotype) in biological family members in order to determine the most likely mode of inheritance for the trait or disease under study.</p>
</dd>
<dt id="term-sensitivity">sensitivity</dt><dd><p>The frequency with which a test yields a true positive result among individuals who actually have the disease or the gene mutation in question. A test with high sensitivity has a low false-negative rate and thus does a good job of correctly identifying affected individuals.</p>
</dd>
<dt id="term-simplex-case">simplex case</dt><dd><p>A single, isolated occurrence of an inherited condition in a family. Simplex cases can result from certain inheritance patterns or be caused by de novo mutations or reduced penetrance within the family. It can also result from nongenetic causes such as adoption or alternate paternity.</p>
</dd>
<dt id="term-single-nucleotide-polymorphism">single nucleotide polymorphism</dt><dd><p>DNA sequence variations that occur when a single nucleotide (adenine, thymine, cytosine, or guanine) in the genome sequence is altered; usually present in at least 1% of the population. Also called SNP.</p>
</dd>
<dt id="term-single-stranded-conformational-polymorphism">single-stranded conformational polymorphism</dt><dd><p>A type of mutation scanning; the identification of abnormally-migrating single-stranded DNA segments on gel electrophoresis. Also called SSCP.</p>
</dd>
<dt id="term-skin-tag">skin tag</dt><dd><p>A small, benign skin growth that may have a stalk (peduncle). Skin tags most commonly appear on the neck, axillary, groin, and inframammary regions. Also called acrochordon.</p>
</dd>
<dt id="term-SNP">SNP</dt><dd><p>DNA sequence variations that occur when a single nucleotide (adenine, thymine, cytosine, or guanine) in the genome sequence is altered; usually present in at least 1% of the population. Also called <strong>single nucleotide polymorphism</strong>.</p>
</dd>
<dt id="term-somatic-variant">somatic variant</dt><dd><p>An alteration in DNA that occurs after conception and is not present within the germline. Somatic variants can occur in any of the cells of the body except the germ cells (sperm and egg) and therefore are not passed on to children. Somatic variants can (but do not always) cause cancer or other diseases.</p>
</dd>
<dt id="term-Southern-blot">Southern blot</dt><dd><p>Electrophoresis-based technique used in genetic testing to detect large deletions in DNA that can be missed by PCR-based genetic testing methods.</p>
</dd>
<dt id="term-specificity">specificity</dt><dd><p>The frequency with which a test yields a true negative result among individuals who do not have the disease or the gene mutation in question. A test with high specificity has a low false-positive rate and thus does a good job of correctly classifying unaffected individuals.</p>
</dd>
<dt id="term-splice-site-mutation">splice-site mutation</dt><dd><p>A genetic alteration in the DNA sequence that occurs at the boundary of an exon and an intron (splice site). This change can disrupt RNA splicing resulting in the loss of exons or the inclusion of introns and an altered protein-coding sequence. Also called splice-site variant.</p>
</dd>
<dt id="term-splice-site-variant">splice-site variant</dt><dd><p>A genetic alteration in the DNA sequence that occurs at the boundary of an exon and an intron (splice site). This change can disrupt RNA splicing resulting in the loss of exons or the inclusion of introns and an altered protein-coding sequence. Also called splice-site mutation.</p>
</dd>
<dt id="term-splicing">splicing</dt><dd><p>The process by which introns, the noncoding regions of genes, are excised out of the primary messenger RNA transcript, and the exons (i.e., coding regions) are joined together to generate mature messenger RNA. The latter serves as the template for synthesis of a specific protein.</p>
</dd>
<dt id="term-sporadic-cancer">sporadic cancer</dt><dd><p>This term has two meanings. It is sometimes used to differentiate cancers occurring in people who do not have a germline mutation that confers increased susceptibility to cancer from cancers occurring in people who are known to carry a mutation. Cancer developing in people who do not carry a high-risk mutation is referred to as sporadic cancer. The distinction is not absolute, because genetic background may influence the likelihood of cancer even in the absence of a specific predisposing mutation. Alternatively, sporadic is also sometimes used to describe cancer occurring in individuals without a family history of cancer.</p>
</dd>
<dt id="term-SSCP">SSCP</dt><dd><p>A type of mutation scanning; the identification of abnormally-migrating single-stranded DNA segments on gel electrophoresis. Also called single-stranded conformational polymorphism.</p>
</dd>
<dt id="term-SSCP-analysis">SSCP analysis</dt><dd><p>A laboratory test used to separate single-stranded nucleic acids based on subtle differences in their DNA sequence, often a single base pair, which results in a different secondary structure and a measurable difference in mobiity through a gel. Also called single-stranded conformation polymorphism analysis.</p>
</dd>
<dt id="term-surveillance">surveillance</dt><dd><p>Periodic clinical evaluation of an individual who is at increased risk of developing a condition (compared with the general population) aimed at detecting new or recurrent disease. In public health, surveillance may also refer to the systematic collection of information regarding the incidence, prevalence, and mortality related to various medical conditions or health-related events.</p>
</dd>
<dt id="term-susceptibility-gene">susceptibility gene</dt><dd><p>A genetic alteration that increases an individual’s susceptibility or predisposition to a certain disease or disorder. When such a variant (or mutation) is inherited, development of symptoms is more likely, but not certain. Also called deleterious mutation, disease-causing mutation, pathogenic variant, and predisposing mutation.</p>
</dd>
<dt id="term-tagging-SNP">tagging SNP</dt><dd><p>A single nucleotide polymorphism, or SNP, that is used to “tag” a particular haplotype in a region of the genome. As a subset of all of the SNPs in the genome, tagging SNPs can be extremely useful for testing the association of a marker locus with a qualitative or quantitative trait locus in that it may not be necessary to genotype all of the SNPs. Also called tagSNP.</p>
</dd>
<dt id="term-tagSNP">tagSNP</dt><dd><p>A single nucleotide polymorphism, or SNP, that is used to “tag” a particular haplotype in a region of the genome. As a subset of all of the SNPs in the genome, tagSNPs can be extremely useful for testing the association of a marker locus with a qualitative or quantitative trait locus in that it may not be necessary to genotype all of the SNPs. Also called tagging SNP.</p>
</dd>
<dt id="term-telomere">telomere</dt><dd><p>The end of a chromosome. Telomeres are made of repetitive sequences of non-coding DNA that protect the chromosome from damage. Telomeres become shorter each time the cell divides.</p>
</dd>
<dt id="term-throughput">throughput</dt><dd><p>The quantity of information, people, or materials that is put through a process in a specific period of time. In medicine, it can be used to describe the efficiency of laboratory procedures, such as genetic sequencing, or the number of patients seen in a clinic in a certain period of time.</p>
</dd>
<dt id="term-transcription">transcription</dt><dd><p>The process of synthesizing messenger RNA (mRNA) from DNA.</p>
</dd>
<dt id="term-translation">translation</dt><dd><p>The process of synthesizing an amino acid sequence (protein product) from the messenger RNA code.</p>
</dd>
<dt id="term-translocation">translocation</dt><dd><p>A type of chromosomal abnormality in which a chromosome breaks and a portion of it reattaches to a different chromosomal location.</p>
</dd>
<dt id="term-trichilemmoma">trichilemmoma</dt><dd><p>A benign tumor arising from the outer cells of the hair follicle.</p>
</dd>
<dt id="term-trinucleotide-repeat">trinucleotide repeat</dt><dd><p>Sequences of 3 nucleotides repeated in tandem on the same contiguous section of chromosome. A certain amount of normal (polymorphic) variation in repeat number with no clinical significance commonly occurs between individuals; however, repeat numbers over a certain threshold can, in some cases, lead to adverse effects on the function of the gene, resulting in genetic disease.</p>
</dd>
<dt id="term-Trio">Trio</dt><dd><p>A trio is a form of a genetic case, consisting of a proband, mother and father.</p>
</dd>
<dt id="term-triple-negative-breast-cancer">triple-negative breast cancer</dt><dd><p>Triple-negative breast cancer is defined by a lack of expression of estrogen receptor (ER), progesterone receptor (PR), and human epidermal growth factor receptor 2 (HER2/neu). Also called ER-negative PR-negative HER2/neu-negative breast cancer.</p>
</dd>
<dt id="term-trisomy">trisomy</dt><dd><p>The presence of an extra chromosome, resulting in a total of three copies of that chromosome instead of the normal 2 copies (e.g., trisomy 21, or Down syndrome).</p>
</dd>
<dt id="term-tumor-sequencing">tumor sequencing</dt><dd><p>Sequencing of somatic tissue, such as tumors, refers to looking for variants in DNA that typically occur after conception. Somatic mutations can occur in any of the cells of the body except the germ cells (sperm and egg) and therefore are not passed on to children. These variants can (but do not always) cause cancer or other diseases.</p>
</dd>
<dt id="term-tumor-suppressor-gene">tumor suppressor gene</dt><dd><p>A type of gene that regulates cell growth. When a tumor suppressor gene is mutated, uncontrolled cell growth may occur. This may contribute to the development of cancer. Also called antioncogene.</p>
</dd>
<dt id="term-unaffected">unaffected</dt><dd><p>An individual who does not manifest symptoms of a condition or disease occurring in his or her family.</p>
</dd>
<dt id="term-unclassified-variant">unclassified variant</dt><dd><p>A variation in a genetic sequence whose association with disease risk is unknown. Also called variant of uncertain significance, variant of unknown significance, and VUS.</p>
</dd>
<dt id="term-uninformative">uninformative</dt><dd><p>A negative test result in an individual where a clearly deleterious mutation has not been found in any family members. The genetic risk status of such an individual must be interpreted in the context of his or her personal and family history. Also called inconclusive and indeterminate.</p>
</dd>
<dt id="term-variable-expression">variable expression</dt><dd><p>Variation in the manner in which a trait is manifested. When there is variable expressivity, the trait may vary in clinical expression from mild to severe. For example, the condition neurofibromatosis type 1 may be mild, presenting with café-au-lait spots only, or may be severe, presenting with neurofibromas and brain tumors.</p>
</dd>
<dt id="term-variant">variant</dt><dd><p>A genetic variant may refer to: A single-nucleotide polymorphism (SNP), in case it is a common genetic variant. A mutation, in a case where it is a rare genetic variant. A copy-number variation.</p>
</dd>
<dt id="term-VUS">VUS</dt><dt id="term-variant-of-unknown-significance">variant of unknown significance</dt><dt id="term-variant-of-uncertain-significance">variant of uncertain significance</dt><dd><p>A variation in a genetic sequence whose association with disease risk is unknown. Also called unclassified variant, variant of uncertain significance, and variant of unknown significance.</p>
</dd>
<dt id="term-WES">WES</dt><dt id="term-whole-exome-sequencing">whole-exome sequencing</dt><dd><p>A laboratory process that is used to determine the nucleotide sequence primarily of the exonic (or protein-coding) regions of an individual’s genome and related sequences, representing approximately 1% of the complete DNA sequence.</p>
</dd>
<dt id="term-WGS">WGS</dt><dt id="term-whole-genome-sequencing">whole-genome sequencing</dt><dd><p>A laboratory process that is used to determine nearly all of the approximately 3 billion nucleotides of an individual’s complete DNA sequence, including non-coding sequence.</p>
</dd>
<dt id="term-X-linked-dominant">X-linked dominant</dt><dd><p>X-linked dominant inheritance refers to genetic conditions associated with mutations in genes on the X chromosome. A single copy of the mutation is enough to cause the disease in both males (who have one X chromosome) and females (who have two X chromosomes). In some conditions, the absence of a functional gene results in the death of affected males.</p>
</dd>
<dt id="term-X-linked-recessive">X-linked recessive</dt><dd><p>X-linked recessive inheritance refers to genetic conditions associated with mutations in genes on the X chromosome. A male carrying such a mutation will be affected, because he carries only one X chromosome. A female carrying a mutation in one gene, with a normal gene on the other X chromosome, is generally unaffected.</p>
</dd>
<dt id="term-Z-score">Z-score</dt><dd><p>A score that indicates how many standard deviations a value is above or below the mean.</p>
</dd>
</dl>
</div>


           </div>
           
          </div>
          <footer>
  

  <hr/>

  <div role="contentinfo">
    <p>
        
        &copy; Copyright 2019, Steve Ayers

    </p>
  </div>
    
    
    
    Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a
    
    <a href="https://github.com/rtfd/sphinx_rtd_theme">theme</a>
    
    provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  

  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>